antonbabenko/pre-commit-terraform
# feat(ci): Build multi-arch Docker images (`amd64`, `arm64`)

Just looking to add ARM builds for my needs



Put an `x` into the box if that apply:


### Description of your changes



Add ARM builds for running on M1/M2 Macs and other Arm CPUs



### How can we test changes

After building, run the following command and check that the output contains `amd64`, `arm` and `arm64`

`docker manifest inspect ghcr.io/antonbabenko/pre-commit-terraform:nightly | jq -r '.manifests[].platform.architecture' | sort -u`
